src/Altinn.Notifications/Program.cs

```diff
@@ -182,6 +186,11 @@ void ConfigureServices(IServiceCollection services, IConfiguration config)
     AddInputModelValidators(services);
     services.AddCoreServices(config);
 
+    services.Configure<Altinn.Common.PEP.Configuration.PlatformSettings>(config.GetSection("PlatformSettings"));
```

Comment:
Should this whole section rather be added to an extension method in the integrations project? Following the pattern for AltinnClients and CoreServices ?